Kernel/Arch/x86/common/Processor.cpp

```diff
@@ -1611,14 +1543,8 @@ extern "C" void enter_thread_context(Thread* from_thread, Thread* to_thread)
     }
 
     auto& processor = Processor::current();
-#if ARCH(I386)
```

Comment:
As part of Kernel: Remove i686 support did you want to move all of Arch/x86/common and Arch/x86/x86_64 into Arch/x86 ?